spyder/plugins/editor/widgets/editor.py

```diff
@@ -863,6 +863,12 @@ def open_symbolfinder_dlg(self):
         self.open_fileswitcher_dlg()
         self.fileswitcher_dlg.set_search_text('@')
 
+    def get_plugin_title(self):
+        """Get the plugin title of the parent widget."""
+        # Needed for the editor stack to use its own fileswitcher instance.
+        # See issue # 9469

```

Comment:
Please change this line to

    # See spyder-ide/spyder#9469

This is the new format to mention issues that we're going to adopt from now on, in order to take advantage of the new URL awareness additions made by @goanpeca.